### PR TITLE
Issue 68: ogm/drivable area are blank in the first few steps of agents

### PR DESCRIPTION
**Pull request by aurorahcx**
- Make sure the render pipeline gets run through after all updates are ready, and before grid maps are returned
Before vs. After (grey squares indicate vehicle positions from observations):
observations

- Set depth bits from 0 to 8 to avoid z-fighting in RGB grid map
Before (ego vehicle is occluded) vs. After (ego vehicle is rendered correctly after the road):
depthbits

### CODE REVIEW

**Comment by aurorahcx** [comment]
Closed the old PR and opened this one to have correct email address linked to my commits for CLA signature

**Review by davidrusu — APPROVED** [review]

File: smarts/core/sensors.py [hunk 1]
```diff
@@ -497,7 +497,9 @@ def _build_offscreen_camera(
         fb_props = FrameBufferProperties()
         fb_props.setRgbColor(True)
         fb_props.setRgbaBits(8, 8, 8, 1)
-        fb_props.setDepthBits(0)
+        # XXX: Though we don't need the depth buffer returned, setting this to 0
+        #      causes undefined behaviour where the ordering of meshes is random.
+        fb_props.setDepthBits(8)
```
Comment (Gamenot) [inline smarts/core/sensors.py:502]: Did a 256 depth buffer fix the issue after all?

**Review by Gamenot — APPROVED** [review]
Cool to see this fixed!

File: smarts/core/sensors.py (hunk 1 above)
Comment (aurorahcx) [reply smarts/core/sensors.py:502]: Yes, I think so. I have attached the screenshots in the comment session.

**Pull request merged**